app/views/shared/_authentication_actions.html.erb

```diff
@@ -1,6 +1,18 @@
-<a href="<%= sign_up_path(state: "new-user") %>" data-tracking-id="ca_feed_home_page" data-tracking-source="feed_card" class="crayons-btn" aria-label="<%= t("views.auth.create.aria_label") %>" data-no-instant>
-  <%= t("views.auth.create.text") %>
-</a>
-<a href="<%= sign_up_path %>" class="crayons-btn crayons-btn--ghost-brand" aria-label="<%= t("views.auth.login.aria_label") %>">
-  <%= t("views.auth.login.text") %>
-</a>
+<div>
+  <div class="align-center">
+    <a href="<%= sign_up_path(state: "new-user") %>" class="crayons-btn w-75" aria-label="<%= t("views.auth.create.aria_label") %>"  data-tracking-id="ca_feed_home_page" data-tracking-source="feed_card" data-no-instant>
+      <%= t("views.auth.create.text") %>
+    </a>
+  </div>
+  <div class="align-center mt-4">
+    <span>
+      <%= t("views.auth.already_have_an_account") %>
+      <a
+        href="/enter"
+        data-tracking-id="si_profile_comment_section_locked"
+        data-tracking-source="comments_locked_cta">
```

Comment:
Question: Can you share a bit the reasoning behind the renaming of the tracking id and source names? I'm wondering why they would change if the card is essentially the same (still within the feed) but with a new design

Reply:
Actually you are right, the id/source should not change. I was working on https://github.com/forem/forem/pull/18635 where it was similar implementation and when I copied code from that I missed changing these. 

Earlier there were tracking-ids for 'Create Account' only but now I have added for `Sign-in` also.

Please check if the new tracking id/source for `sign_in` looks good.

Thanks for picking this.